index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+    integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+    crossorigin="anonymous" referrerpolicy="no-referrer" />
+   <link rel="stylesheet" href="style.css">
+  <title>Personal Portfolio</title>
+</head>
+<body>
+   <header>
+      <nav>
+        <div class="logo"><a href="#">I<span>.J</span></a></div>
+         <div class="hamburger-menu"><i class="fa-solid fa-bars"></i></div>
+      </nav>
+   </header>
+
+   <section class="title">
+     <h1>I'm Isaac. Glad To see you!</h1>
+
+     <p>
+      I'm a Software developer! Can help you build a product, feature or Website Look through Some of my Work and experience! if you like What you see and have a project you need coded, don't hesitate to contact me.
+     </p>
+
+    <span class="span">LET'S CONNECT</span>
```

Comment:
- [ ] Please making use of an `hx` tag such as `h1,h2` etc to display this element on the web browser :+1: It is  important to make use of semantic tags rather than using the non semantic `span` tag :+1: